tests/syscall_socketcall/test

```diff
@@ -81,7 +81,7 @@ my $found_sockaddr   = 0;
 while ( $line = <$fh_out> ) {
     if ( $line =~ /^type=SOCKADDR / ) {
         if ( $line =~
-/ : saddr=(\{ fam=|)inet (laddr=|host:)127.0.0.1 (lport=|serv:)$portno( \}|)/
+/ : saddr=(\{ fam=|\{ saddr_fam=)inet (laddr=|host:)127.0.0.1 (lport=|serv:)$portno( \}|)/
```

Comment:
The original regex accepted also an empty string before `inet` (` : saddr=inet[...]`) - is it intended that the new one accepts only ` : saddr={ fam=inet[...]` or ` : saddr={ saddr_fam=inet[...]`?

Reply:
Ah, good catch! No, it is a mistake, I'll fix it. Thanks!

Reply:
Done.